bin/generators/credentials/list.js

```diff
@@ -6,33 +6,37 @@ module.exports = class extends eg.Generator {
 
     this.configureCommand({
       command: 'list [options]',
-      description: 'List all credentials for Consumer (User or App)',
+      description: 'List Consumer (User or App) credentials',
       builder: yargs =>
         yargs
           .usage(`Usage: $0 ${process.argv[2]} list [options]`)
           .example(`$0 ${process.argv[2]} list -c 7498d1a9-7f90-4438-a9b7-0ba4c6022353`)
-          .string(['c'])
+          .string(['c', 'f'])
           .describe('c', 'Consumer ID: can be User ID or username or app ID')
+          .describe('f', 'Comma separated list of credential state (active, archived), default: active')
           .alias('c', 'consumerId').nargs('c', 1)
-          .group(['c'], 'Options:')
+          .alias('f', 'filter')
+          .group(['c', 'i'], 'Options:')
```

Comment:
Note to self: what's the `i` option for

Reply:
see comment in slack, feel free to implement differently